test/scripts/integration_tests.py

```diff
@@ -74,133 +74,6 @@ def delete(self, filepath, raise_error=False):
         """
         self.__run_command("delete", filepath, raise_error)
 
-
-class Helm(object):
-    """
-    Class used to interact with helm and return data
-    """
-
-    def delete(self, release_name):
-        """
-        Delete a named helm release
-        """
-        subprocess.call(
-            [
-                "helm",
-                "delete",
-                "--purge",
-                release_name
-            ]
-        )
-
-    def init(self):
-        """
-        Run a helm init to ensure there is a working tiller
-        :return:
-        """
-        subprocess.check_call(
-            [
-                "helm",
-                "init"
-            ]
-        )
-
-    def status(self, release_name):
-        """
-        Return the output of helm status
-        :param release_name: The name of the release to get
-        :return: The output of command as bytes
-        """
-        try:
-            output = subprocess.check_output(
-                [
-                    "helm",
-                    "status",
-                    release_name
-                ]
-            )
-            return output
-        except subprocess.CalledProcessError as error:
-            return error.output
-
-
-class HelmIntegrationTest(TestCase):
```

Comment:
Makes sense to take out this test, but it seems like something should replace it. That's just a rough idea since I haven't put much thought into what the new one should be.